backend/tracim_backend/lib/search/elasticsearch_search/elasticsearch_search.py

```diff
@@ -536,6 +627,143 @@ def search_content(
         res.search_fields = search_fields
         return res
 
+    def search_user(
+        self,
+        search_string: str,
+        search_fields: typing.List[str] = DEFAULT_USER_SEARCH_FIELDS,
+        workspace_ids: typing.Optional[typing.List[int]] = None,
+        newest_authored_content_date_from: typing.Optional[datetime] = None,
+        newest_authored_content_date_to: typing.Optional[datetime] = None,
+        show_deleted: bool = False,
+        show_active: bool = True,
+        page_nb: int = 0,
+        size: int = 10,
+    ) -> UserSearchResponse:
+        search = Search(
+            using=self.es,
+            doc_type=self.IndexedUser,
+            index=self._get_index_parameters(self.IndexedUser).alias,
+        )
+        fields = [
+            "public_name.exact^5",
+            "username.exact^5",
+            "public_name^3",
+            "username^3",
+            "custom_properties",
+        ]
+
+        search_fields = search_fields or DEFAULT_USER_SEARCH_FIELDS
+        fields = [field for field in fields if name_starts_with_any_prefix(field, search_fields)]
```

Comment:
I do suggest to follow similar solution as content for this, using enum:
https://github.com/tracim/tracim/pull/4198/commits/c7622b309b6e072d4d53173cde97917b3066a492
I don't have strong opinion if we should fix this now or later, but i expect we try to have some consistency for this.

Reply:
Done.